example/package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.0",
     "web-vitals": "^0.2.4",
+    "wrld-react": "^0.1.2261",
     "wrld.js": "^0.1.2255"
```

Comment:
Can we do without a dependency on wrld.js, as it's already a dependency of wrld-react (and it saves having two version numbers specified explicitly)

Reply:
Forgot to remove it. Done now